### PR TITLE
Handle non-sorted lists in dicts when comparing dicts

### PR DESCRIPTION
**Pull request by berland**
This makes the printed diff from the sanity checker make sense

Solves #481

### CODE REVIEW

**Comment by jonathan-eq** [comment]
@berland How does the sanity checker output look after this commit?

File: komodo/symlink/sanity_check.py [hunk 1]
```diff
@@ -91,10 +91,21 @@ def assert_root_nodes(link_dict):
         )
 
 
-def _compare_dicts(dict_1, dict_2):
+def _sort_lists_in_nested_dicts(_dict: dict) -> dict:
+    result = {}
+    for key, value in _dict.items():
+        if isinstance(value, list):
+            result[key] = sorted(value)
+        else:
```
Comment (jonathan-eq) [inline komodo/symlink/sanity_check.py:99]: Will there ever be a situation where there is another layer of dict? Should we check for this, and use this function recursively?

**Review by jonathan-eq — CHANGES_REQUESTED** [review]

File: komodo/symlink/sanity_check.py (hunk 1 above)
Comment (berland) [reply komodo/symlink/sanity_check.py:99]: I will remove `nested` from the function name. It was originally a recursive sorter, but that was before I understood the sorting thing was about lists..

**Review by jonathan-eq — APPROVED** [review]
LGTM!

**Pull request merged**